pkg/model/image_manager_test.go

```diff
@@ -31,3 +31,15 @@ func TestImageManager_test_defining_image_using_environment_variable(t *testing.
 	//then
 	assert.Equal(t, "test", imageChooser.Images[KeycloakImage])
 }
+
+func TestImageManager_test_overriding_rhsso_image_using_environment_variable(t *testing.T) {
```

Comment:
I think this test won't work with operator running in cluster, am I correct? The env var would have to be set on the cluster, not on local machine.

Reply:
Yes. This is a unit test.

Reply:
Ah, right, sorry. :D